mlflow/models/evaluation/utils/trace.py

```diff
@@ -123,6 +123,16 @@ def _setup_autolog(module):
                         AUTOLOGGING_INTEGRATIONS.pop(flavor, None)
                 except ImportError:
                     pass
+                except Exception as e:
+                    if original_config is None or (
```

Comment:
the else branch is `disable=True or silent=True`, wondering what exception could be thrown.

Reply:
Usually it won't raise exception in that case, otherwise it is bugs inside autologging implementation. but adding exception capturing is still useful in the case , to avoid `evaluate` invocation being interrupted by unrelated autologging bug.